test/e2e/pkg/testnet.go

```diff
@@ -124,6 +124,7 @@ func LoadTestnet(file string) (*Testnet, error) {
 		Validators:       map[*Node]int64{},
 		ValidatorUpdates: map[int64]map[*Node]int64{},
 		Nodes:            []*Node{},
+		Evidence:         int(manifest.Evidence),
```

Comment:
What is the type of `manifest.Evidence` and do we really need to cast it? Why not make the root type `int`?

Reply:
it's int64. I will check if the manifest works with directly using `int`